to push to week branch

Naol19 opened the pull request:
this is my first progress on the project I am doing and, in this progress(step) I have created three class Person, Student, Teacher and also create a Geter and Seter method on some of my class. and the main and interesting part in this code is I implemented inheritance on student and teacher class from the person class.

kueth123 committed:
to push to week branch

Benawi commented on Person.java:
```diff
@@ -0,0 +1,52 @@
+public class Person {
+    public int id;
+    public String name;
+    public int age;
```
- Modifier not requested in this task

Benawi commented on Person.java:
```diff
@@ -0,0 +1,52 @@
+    public boolean parent_permission;
+    //constructor
+
+    public Person(int id, String name, int age, boolean parent_permission) {
```
- Default value not initialized for name and parent_permission

Benawi commented on Student.java:
```diff
@@ -0,0 +1,13 @@
+public class Student extends Person {
+    public String getClassroom() {
+        return classroom;
+    }
+    public void setClassroom(String classroom) {
+        this.classroom = classroom;
+    }
```
- getter and setter method not requested here

Benawi commented on Teacher.java:
```diff
@@ -0,0 +1,15 @@
+        this.specialization = specialization;
+    }
+
+    public Teacher(int id, String name, int age, boolean parent_permission) {
+        super(id, name, age, parent_permission);
+    }
```
- Not required

Benawi reviewed (changes requested):
#### Hi @Naol19 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
You did a great job, keep up the good work 👍🏼

![giphy](https://user-images.githubusercontent.com/25227022/169391888-8dd8108f-0d42-4e54-87cc-a3f9fd392dbc.gif)


## Highlights

✔️  Create Person Class 
✔️  Create Student Class 
✔️  Create Teacher Class 

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. @benawi

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

Naol19 committed:
Update Person.java

I have removed the modifier I have mentioned before in the person class and also I have initialized the default value

Naol19 committed:
Update Student.java

I have removed the getter and setter method on the student class

Naol19 committed:
Update Teacher.java

I have removed the constructor that I have created twice

Benawi reviewed (approved):
Hi @Naol19 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Congratulations! 🎉 

![Alt Text](https://media.giphy.com/media/JoSiL3L4ON8DlyVxv0/giphy.gif)
## Highlights
✔️ Create Person Class
✔️ Create Student Class
✔️ Create Teacher Class
✔️ PR is well descriptive 👍 
✔️ Requested changes implemented 👍
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you  take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@benawi** in your question so I can receive the notification.

------
